### DISCUSSION TITLE
problema en la construcción de MVE

### DISCUSSION THREAD

**Discussion by dectun** [Homebrew/discussions#3172] [Getting started]
https://github.com/simonfuhrmann/mve

Al intentar instalar MVE en macbook air M1 de 16gb de ram. Me da el siguiente error cuando voy a dar el paso de -j8;

clangclang: clang: : error: error: error: the clang compiler does not support '-march=native'the clang compiler does not support '-march=native'the clang compiler does not support '-march=native'

clang: 
error: the clang compiler does not support '-march=native'
make[2]: *** [system.o] Error 1
make[2]: *** Waiting for unfinished jobs....
make[2]: *** [ini_parser.o] Error 1
make[2]: *** [file_system.o] Error 1
make[2]: *** [arguments.o] Error 1
make[1]: *** [all] Error 2
make: *** [all] Error 2

**Comment by carlocab** [comment]
This is not Homebrew related. Please check https://github.com/simonfuhrmann/mve for solutions.
